client/src/document.js

```diff
@@ -137,21 +137,49 @@ function SidebarLeaf({ title, content }) {
 
 function SidebarLeaflets({ node }) {
   return (
-    <details>
-      <summary>{node.title}</summary>
-      <ol>
-        {node.content.map(childNode => {
-          return (
-            <li key={childNode.uri}>
-              <Link to={childNode.uri}>{childNode.title}</Link>
-            </li>
-          );
-        })}
-      </ol>
-    </details>
+    <Location>
+      {({ location }) => {
+        const activeChild = getChildActiveInCurrentLocation(
```

Comment:
It's not wrong but why move this to a top-level function on its own? 
After all, it's just 3 lines. Or 1 if you'd kept it really short :)
```javascript
const activeChild = content.find(n => n.uri === location.pathname);
```

I'm just concerned that it escalates the amount of "noise" since the function is never really re-used nor unit tested (not that I think it should be either).

Reply:
Sure no problem! Initially I made this a function call because I was expecting it to do something complicated, but after I finished the implementation, it turned out to be quite simple 😄